Includes/Rosetta/Tasks/SimpleTasks/RandomCardTask.hpp

```diff
@@ -13,26 +13,28 @@ namespace RosettaStone::SimpleTasks
 class RandomCardTask : public ITask
 {
  public:
+    //! Constructs task with given \p entityType and \p opposite.
+    //! \param entityType The entity type to choose the random card from.
+    //! \param opposite The flag that indicates the card is for the opponent.
+    RandomCardTask(EntityType entityType, bool opposite = false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class 'RandomCardTask' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4554843)